src/frontends/tensorflow_common/src/CMakeLists.txt

```diff
@@ -3,20 +3,20 @@
 #
 
 set(TARGET_NAME "openvino_tensorflow_common")
-
 get_filename_component(root_dir "${CMAKE_CURRENT_SOURCE_DIR}" DIRECTORY)
 
 file(GLOB_RECURSE LIBRARY_SRC ${root_dir}/src/*.cpp)
 file(GLOB_RECURSE LIBRARY_HEADERS ${root_dir}/src/*.hpp ${root_dir}/dev_api/*.hpp)
 file(GLOB_RECURSE LIBRARY_PUBLIC_HEADERS ${root_dir}/include/*.hpp)
+
 add_library(${TARGET_NAME} STATIC ${LIBRARY_SRC} ${LIBRARY_HEADERS} ${LIBRARY_PUBLIC_HEADERS})
-add_library(openvino::frontend::tensorflow_common ALIAS ${TARGET_NAME})
 target_link_libraries(${TARGET_NAME} PRIVATE openvino::util)
+add_library(openvino::frontend::tensorflow_common ALIAS ${TARGET_NAME})
 
 add_library(${TARGET_NAME}_dev INTERFACE)
-add_library(openvino::frontend::tensorflow_common::dev ALIAS ${TARGET_NAME}_dev)
 target_link_libraries(${TARGET_NAME}_dev INTERFACE ${TARGET_NAME})
 target_include_directories(${TARGET_NAME}_dev INTERFACE $<BUILD_INTERFACE:${root_dir}/dev_api>)
+add_library(openvino::frontend::tensorflow_common::dev ALIAS ${TARGET_NAME}_dev)
```

Comment:
I suppose `openvino::frontend::tensorflow_common` is dev by its meaning, is not it?
what is the difference between headers from `dev_api` and `include` folders?

Reply:
include is for the users. dev_api for frontends

Reply:
but you don't install headers in `include` :) and don't export their API

Reply:
It looks like I do now) please check if the question is obsolete now.